.ci_support/linux_64_cuda_compiler_version10.0numpy1.16python3.6.____cpython.yaml

```diff
@@ -35,8 +35,7 @@ target_platform:
 zip_keys:
 - - c_compiler_version
   - cxx_compiler_version
-- - cdt_name
```

Comment:
May want to manually add this back in. Also is missing in the other variant files